plugins/org/src/components/Cards/Group/MembersList/MembersListCard.tsx

```diff
@@ -105,7 +107,7 @@ const MemberComponent = (props: { member: UserEntity }) => {
               <Link
                 data-testid="user-link"
                 to={generatePath(
-                  `/catalog/:namespace/user/${metaName}`,
+                  `${catalogLink()}/:namespace/user/${metaName}`,
```

Comment:
I know this is a Draft but could we have used the `EntityRefLink` component for this instead? 

https://github.com/backstage/backstage/blob/999376ee010c59edf51074f53d21a4acd4d3404d/plugins/catalog-react/src/components/EntityRefLink/EntityRefLink.tsx#L47

Reply:
That was the first thing I tried but the EntityRefLink returns the entity in this format: user:<name> (not great looking). Then once I saw that the catalogIndexRouteRef was used in another place I just went with that. But I can look into it further and see if there is a way to make it work with EntityRefLink if you think that would be more optimal.

Reply:
It just seems like the perfect use for it. Have you tried with the `title` prop? Here's an example:

https://github.com/backstage/backstage/blob/999376ee010c59edf51074f53d21a4acd4d3404d/plugins/catalog/src/components/CatalogTable/columns.tsx#L54-L58

Reply:
Your right, that did it. My bad, I didn't realize the title prop replaced the value that was being displayed by the EntityRefLink. I guess that's what I get for not trying it out in the first place😅. I will do some corrections and then push the new version for review.

Reply:
That's great, no worries, thanks for taking on the work of fixing this 👍

Reply:
EntityRefLink has a defaultKind prop, that you can set to "user". That should make the prefix disappear (for users :)).